src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -183,6 +181,12 @@ public static function call(callable $function, ...$arguments)
 
     public function __construct(?BufferingLogger $bootstrappingLogger = null, bool $debug = false)
     {
+        // BC layer for PHP < 8.4
+        if (\defined('E_STRICT')) {
```

Comment:
```suggestion
        if (\PHP_VERSION_ID < 80400) {
```

as I read the linked RFC the constant won't be removed in PHP 8.4

Reply:
There's also this sentence:

> It should be noted that in theory external extensions could still emit a E_STRICT level error, but this seems unlikely.

Are we sure we want to completely remove support from PHP 8.4 onwards? We could keep it just in case, and use the raw value in the table to avoid depreciation until the next major release. Just a thought, your solution works for me too.

Reply:
As I read the RFC the error level will be removed and trying to emit a strict error would fail.

Reply:
Actually, it seems it will become no-op, but the constant remains: https://github.com/php/php-src/pull/13053#issuecomment-1873311871 for now. But indeed, if it becomes a no-op, let's apply your suggestion